### ISSUE TITLE
Blocks: move design tools to shared Extensions utils package

### ISSUE THREAD

**Issue by mashikag** [Automattic/jetpack#24956]
Move those tools to the new @automattic/jetpack-shared-extension-utils in a follow-up PR. 

https://github.com/Automattic/jetpack/pull/22694#issuecomment-1035968239

**Comment by mashikag** [comment]
@yansern can you please have a look at this whenever possible? This is taken from a draft note from `Apex: Inbox` board. Perhaps this was already done? If not can you add a bit more into description. It is not very clear as to what is to be moved to the `@automattic/jetpack-shared-extension-utils` package.

**Comment by mashikag** [comment]
@yansern can you have a look at this? It is hard to understand what is to be done. 
![Screen Shot 2022-10-18 at 10 21 09](https://user-images.githubusercontent.com/2019970/196376608-e64b5c13-6681-4c47-8ddf-6733ae34a26c.png)
https://github.com/Automattic/jetpack/pull/22694#issuecomment-1035968239
What is `this`? 😄  Me and @TimBroddin looked at the PR changes but we cannot find any tool code that should be moved to shared utils.

**Comment by yansern** [comment]
The component was InspectorNotice. And looks like it was already moved into shared components.
https://github.com/Automattic/jetpack/commit/4c794302f128f891b52731cacf0cdcf75bbc45f6

As for the better translation functions, I believe we settled with `_n()`. https://github.com/Automattic/jetpack/pull/22694#discussion_r804768029

I think this issue can be closed!
